docs/using-est-with-certd.md

```diff
@@ -0,0 +1,169 @@
+# Using EST with Certificates Service
+
+This document covers how to configure Certificates Service to issue certificates over EST.
+
+## Prerequisites
+
+- Certificates Service and Keys Service configured and running
+- A working EST server that processes enrollment requests
+
+## Configuration Options
+
+EST certificate issuance is configured in Certificates Service's config.toml, which by default is `/etc/aziot/certd/config.toml` or a file in `/etc/aziot/certd/config.d/`.
+
+The global `[cert_issuance.est]` section controls options that apply to all certificates issued over EST.
+
+Each certificate has a `[cert_issuance]` entry. By specifying `method = "est"`, the corresponding certificate will be issued over EST.
+
+```toml
+# Global options that affect all EST-issued certificates.
+[cert_issuance.est]
+
+# Trusted root certificates to validate the EST server's TLS certificate;
+# optional depending on how the EST server is configured.
+# It is not required for servers with a publicly-rooted TLS certificate.
+trusted_certs = ["cert-id"]
+
+# Provides a default URL if the EST URL is not provided for a certificate.
+# Optional if each certificate issuance specifies a URL.
+[cert_issuance.est.urls]
+default = "https://est.example.com/.well-known/est"
+
+# Sample configuration of a single EST-issued certificate.
+# Replace `name` with the desired certificate name.
+[cert_issuance.name]
+
+# Identifies this certificate as being issued over EST.
+method = "est"
+
+# Optional certificate common name. Defaults to the CSR's common name if not provided.
+common_name = "common name"
+
+# Optional number of days between certificate issuance and expiry. Defaults to 30 if not provided.
+expiry_days = 30
+
+# Optional EST URL to issue this certificate. Defaults to the `default` URL in `[cert_issuance.est.urls]`
+# if not provided. The URL must be provided either here or in default, i.e. certd will fail if no URL is
+# provided here and no default exists.
+url = "https://est.example.com/.well-known/est"
+
+# Below are options for authenticating with the EST server. The required options will depend on the EST
+# server's configuration.
+
+# Authentication with TLS client certificate. Provide the cert ID of the client cert and its corresponding
+# private key. Note that the aziotcs user must be authorized to access `identity_pk` in Keys Service.
+identity_cert = "identity-cert-id"
```

Comment:
Would it be better to document these in the `[cert_issuance.est]` section instead of the per-cert section? I assume most users will have a single server with the same auth, and the only thing that's per cert is the URL.

Reply:
Makes sense. I moved these under the global section but left a note under cert_issuance that it's possible to configure separately.